### ISSUE TITLE
Capell-1956-XX: Basic wordlist on Australian languages

### ISSUE THREAD

**Issue by LinguList** [clld/concepticon-data#371]
Found this through and article by Nick Evans, the list seems to list some 40+ items which are "basic" with respect to Australian languages.

**Comment by DavidNash1** [comment]
Capell 1956:85, with his proposed Australian proto-forms, and identification number of Swadesh 200 list (where possible)
[Capell1956.xlsx](https://github.com/clld/concepticon-data/files/2532053/Capell1956.xlsx)

**Comment by Kristina-Pianykh** [comment]
@DavidNash1 is there any other way I could download this list? I'm not sure that GitHub allows access to directly attached files.

**Comment by DavidNash1** [comment]
> @DavidNash1 is there any other way I could download this list? I'm not sure that GitHub allows access to directly attached files.

When I click on the link (https://github.com/clld/concepticon-data/files/2532053/Capell1956.xlsx) the spreadsheet downloads (but doesn't display in the browser).
This message composition pane at the bottom invites "Attach files by ..." so GitHub presumably allows some kind of access to files so attached.

**Comment by SimonGreenhill** [comment]
I've just emailed it to you @Kristina-Pianykh , thanks @DavidNash1!

**Comment by DavidNash1** [comment]
> I've just emailed it to you @Kristina-Pianykh , thanks @DavidNash1!

Goodoh; and did you get it from GitHub here?!

**Comment by SimonGreenhill** [comment]
yes, but it bounced through a bunch of URLS and took quite a while, so maybe I just got lucky?

**Comment by Kristina-Pianykh** [comment]
It worked this time, thank you @DavidNash1 and @SimonGreenhill!
